bundle/downstream/manifests/quay-bridge-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,513 @@
+      name: quayintegrations.quay.redhat.com
+      specDescriptors:
+      - description: AllowlistNamespaces is a list of namespaces to include
+        displayName: List of namespaces to include
+        path: allowlistNamespaces
+      - description: ClusterID refers to the ID associated with this cluster.
+        displayName: Cluster ID
+        path: clusterID
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: CredentialsSecret refers to the Secret containing credentials
+          to communicate with the Quay registry.
+        displayName: Credentials secret
+        path: credentialsSecret
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes:Secret
+      - description: Key represents the specific key to reference from the secret
+        displayName: Key within the secret
+        path: credentialsSecret.key
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: Name represents the name of the secret
+        displayName: Name of the secret
+        path: credentialsSecret.name
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: Namespace represents the namespace containing the secret
+        displayName: Namespace containing the secret
+        path: credentialsSecret.namespace
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: DenylistNamespaces is a list of namespaces to exclude.
+        displayName: List of namespaces to exclude
+        path: denylistNamespaces
+      - description: InsecureRegistry refers to whether to skip TLS verification to
+          the Quay registry.
+        displayName: Insecure Registry
+        path: insecureRegistry
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+      - description: OrganizationPrefix is the prefix assigned to organizations.
+        displayName: Organization Prefix
+        path: organizationPrefix
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: QuayHostname is the hostname of the Quay registry.
+        displayName: Quay hostname
+        path: quayHostname
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: ScheduledImageStreamImport determines whether to enable import
+          scheduling on all managed ImageStreams.
+        displayName: Schedule ImageStream Imports
+        path: scheduledImageStreamImport
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+      statusDescriptors:
+      - displayName: Conditions
+        path: conditions
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.conditions
+      - displayName: Last Updated Time
+        path: lastUpdate
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      version: v1
+  description: |-
+    ## Overview
```

Comment:
@ricardomaraschini None of the manual webhook setup is needed anymore. Most of the content in the `description` field were taken from the original README. Since the v1.x alignment with operator-sdk, these steps are no longer needed

Reply:
@harishsurf We need to remove the manual steps from this `description` field.